### DISCUSSION TITLE
More detail on self hosting

### DISCUSSION THREAD

**Discussion by tsadarsh** [github/india#15] [Open Source] [answered]
I'm watching Journey of a project made in India: Hoppscotch. What does it meant to self host projects?

**Comment by punitm0** [comment]
self hosting means hosting on your own infrastructure

**Comment by AnandChowdhary** [comment, marked as answer]
When you're an enterprise or a large company, you might want to host your own instance of a project, perhaps because you want your data to remain on your own servers, or you have a specific bandwidth need. This is a common practice for open-source projects that offer a publicly available hosted version. For example, you can use [sourcegraph/sourcegraph](https://github.com/sourcegraph/sourcegraph) by going to their website on https://sourcegraph.com, or you can [host it yourself](https://docs.sourcegraph.com/#quickstart-guide) on your own infrastructure.

Some other examples of self-hosted open-source tools I use every day include [getsentry/sentry](https://github.com/getsentry/sentry), [elastic/elasticsearch](https://github.com/elastic/elasticsearch), and [caprover/caprover](https://github.com/caprover/caprover). You should have a look at [awesome-selfhosted/awesome-selfhosted](https://github.com/awesome-selfhosted/awesome-selfhosted), which has a long list of products in this space.

**Reply by sidharthachatterjee** [reply]
Typically means running on your own infrastructure.
